src/libcore/sync/atomic.rs

```diff
@@ -140,10 +140,15 @@ unsafe impl<T> Sync for AtomicPtr<T> {}
 /// to be moved either before or after the atomic operation; on the other end
 /// "relaxed" atomics allow all reorderings.
 ///
+/// If you are confused or don't have enough time to research which ordering to use, use `SeqCst`.
+/// Of all the options it has the most unsurpising effect (see the nomicon for details. [1]) 
+/// The downside is you miss out on several optimizations the other orderings offer.
+///
```

Comment:
I’d argue that if one is confused/doesn’t have time to research, they should not be writing code involving atomics at all and just use a `Mutex`.